h2o-algos/src/main/java/hex/schemas/StackedEnsembleV99.java

```diff
@@ -64,8 +65,11 @@
             help = "Keep level one frame used for metalearner training.")
     public boolean keep_levelone_frame;
 
-    @API(help = "Parameters for metalearner algo", direction = API.Direction.INOUT)
+    @API(help = "Parameters for metalearner algorithm", direction = API.Direction.INOUT)
     public String metalearner_params;
+
+    @API(help = "Seed for metalearner algorithm", gridable = true)
```

Comment:
Let's change this to `"Seed for random numbers; passed through to the metalearner algorithm. Defaults to -1 (time-based random number)".`